chall01/dmukaliy.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+import sys
+
+def error():
+	sys.stdout.write("usage: ./dmukaliy.py <a-zA-Z string>\n")
```

Comment:
you should avoid hardcoding your script name, you can use `sys.argv[0]` to find the script name